README.md

```diff
@@ -1,13 +1,66 @@
-# Shopgate Connect - Extension Boilerplate
+# Shopgate Connect - Extension Product Image badges
 
-This boilerplate contains all important folders and files which are required for an extension.
+Extension will allow a merchant to display a image badge on a product image based on tag or property. If you wish to use a property then you must use extension [proudcts-add-properties](https://github.com/shopgate/ext-products-add-properties) in order to have the property exposed in the same location as tags.
 
-## Steps to start with the boilerplate
+## Configuration
+There are two required configurations for this extension.
 
-- Replace the `id` property in the file `extension-config.json` with the id of your extension
-- Replace the `version` property in the file `extension-config.json` with version of your wish
-- If you don't need a pipeline, remove the `awesomeOrganization.awesomePipeline.v1.json` in the `pipelines` folder
-- If you want to use a pipelines, modify the `awesomeOrganization.awesomePipeline.v1.json` in the `pipelines` folder and rename it to the new name 
+#### badgeMap: 
+The configuration badgeMap will be used to determine which product tags or properties are used to denote the need for the associated badge to be added to the product image
+Default Values:
+```json
+{
+  "badgeMap": [
+    {
+      "tags": [],
```

Comment:
I think this is now called triggerTags. I also thing an example tag should be added.